server.go

```diff
@@ -782,6 +785,18 @@ func (sp *serverPeer) enforceNodeBloomFlag(cmd string) bool {
 	return true
 }
 
+func (sp *serverPeer) OnFeeFilter(p *peer.Peer, msg *wire.MsgFeeFilter) {
```

Comment:
Given it's still a WIP, I would guess @dajohi just hasn't tied it in yet, but there will need to be an entry in `newPeerConfig` to register this handler to the peer [here](https://github.com/btcsuite/btcd/blob/83432785f3df1f0b6f8e6926dfe0987a34ea5b3c/server.go#L1530).